Migrations/widgets.recipe.json

```diff
@@ -765,6 +809,56 @@
               }
             }
           ]
+        },
+        {
+          "Name": "Card",
+          "Settings": {},
+          "ContentPartFieldDefinitionRecords": [
+            {
+              "FieldName": "MediaField",
+              "Name": "Image",
+              "Settings": {
+                "DisplayName": "Image",
+                "Position": "0"
+              }
+            },
+            {
+              "FieldName": "TextField",
```

Comment:
@peterkeating Why are the CTA Label and CTA Url different fields when this could be done with a Link Field?

Reply:
Didn't know there was a link field. Does the URL field within the link field allow for relative URLs?

Reply:
Had a play with the `LinkField` and it doesn't support relative/hash URLs.